### DISCUSSION TITLE
Downgrade is not working in foundry for openzeppelin

### DISCUSSION THREAD

**Discussion by veebull** [Cyfrin/foundry-full-course-cu#1027] [Q&A] [answered]
I am on UUPS lesson and the latest OZ upgradebale library is not working for this lesson. Function ```upgradeTo``` is not exist.
So I've tried to downgrade for other specific version
 ```forge install OpenZeppelin/openzeppelin-contracts-upgradeable@4.8.3 --no-commit```
 It does not work.
 Error is 

 ```Tag: "4.8.3" not found for repo "https://github.com/OpenZeppelin/openzeppelin-contracts-upgradeable"!```

Please help me!

**Comment by 0xusmanf** [comment, marked as answer]
Try the following command `forge install OpenZeppelin/openzeppelin-contracts-upgradeable@v4.8.3 --no-commit`. You have missed writing `v` while specifying the tag.
